src/lib/rnp.cpp

```diff
@@ -60,6 +60,7 @@
 #include <botan/secmem.h>
 #include "ffi-priv-types.h"
 #include "file-utils.h"
+#include "cmdline-utils.h"
```

Comment:
Why do we need cmdline-utils.h here? We do not process command line parameters on FFI/library layer.

Reply:
You're right, this should be removed.